src/common/transformations/src/transformations/common_optimizations/fuse_rotary_positional_embeddings.cpp

```diff
@@ -606,10 +618,15 @@ ov::pass::RoPEFusionQwen::RoPEFusionQwen(int split_output_id) {
         makePattern<opset1::Squeeze>({ListUnpack_586_Split->output(0), -2});  //  tensor_array<f32[?,?,32,64]>
     auto cat_Concat_593 = makePattern<opset1::Concat>({ListUnpack_586_Squeeze_0, ListUnpack_586_Squeeze},
                                                       {{"axis", -1}});  //  tensor_array<f32[?,?,32,128]>
+    auto slice_StridedSlice_470 = GenStridedSlice(rotary_emb_sin,
+                                                  ScatterUpdate_463814,
+                                                  {0, INT_MAX},
+                                                  {1, 1},
+                                                  1);  //  tensor_array<f32[1,..4096,1,128]>
     auto slice_Slice_470 =
-        GenSlice2(rotary_emb_sin, ScatterUpdate_463814, {0, INT_MAX}, {1, 1}, 1);  //  tensor_array<f32[1,..4096,1,128]>
+        makePattern<opset8::Slice>({rotary_emb_sin, Gather_377635 | neg_Multiply, {INT_MAX}, {1}, {1}});
```

Comment:
ScatterUpdate_463814 is only needed for StridedSlice to set a value to a specific position (e.g. 2) because we can't provide a specific axis (axis = 2) to StridedSlice, we need to specify all begin/end input values.
From my analysis, I didn't see any ScatterUpdate ops connected to Slice after changing generation to Slice in MOC transformation pipeline.